app/components/forms/song/LineFields/LineFields.tsx

```diff
@@ -44,41 +47,68 @@ export const LineFields: React.FC<Props> = ({
     placement: "left",
   });
 
-  const insertMark = (index: number) => {
+  function insertMark(index: number) {
```

Comment:
Stylistic changes to make it easier to discern in the component what is a variable and what is a function